packages/ripple-binary-codec/src/utils.ts

```diff
@@ -0,0 +1,86 @@
+import TypedArray = NodeJS.TypedArray
+
+export function writeUInt8(
```

Comment:
TODO: Clean up and document.

These were helpful methods that lived directly on the `Buffer` object.

Reply:
DataView ?

Reply:
It is the inner type of a UInt array.  Usefully for working with the different implementations of a TypedArray